README.md

```diff
@@ -345,6 +346,48 @@ To capture data from EventSources running in the same process as EventFlow, the
 
 (*) Either providerName, or providerGuid must be specified. When both are specified, provider GUID takes precedence.
 
+#### Application Insights input
+
+*Nuget package:* [**Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelementryProcessor**](https://www.nuget.org/packages/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/)
+
+Application Insights input is designed for the following scenario:
+1. You have an application that uses Application Insights for monitoring and diagnostics.
+2. You want to send a portion of your Application Insights telemetry to some destination other than Application Insights (e.g. Azure EventHub or Elasticsearch; the assumption is there is an EventFlow output for where the data needs to go).
```

Comment:
Great idea! Added the following sentence:

For example, you might want to leverage [Application Insights sampling capabilities](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-sampling) to reduce the amount of data analyzed by Application Insights without losing analysis fidelity, while sending full raw logs to Elasticsearch to do detailed log search during problem troubleshooting.